docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+
+Default: `{<<palette_fn>>}`
+
+|`font`
+|`style`
+|The legend and tick mark fonts 
+
+Default: `{<<font_fn>>}`
+
+|`legend`
+|`string`, `boolean`
+|The legend position, `"nw"`, `"sw"`, `"ne"`, `"se"`, or `false` 
+
+Default: `"ne"`
+
+|`yaxis`
+|`boolean`, `axisConfig`
+|Axis configuration, or use `false` to disable 
+
+Default: `true`
+
+|`xaxis`
+|`boolean`, `axisConfig`
+|Axis configuration, or use `false` to disable 
+
+Default: `true`
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[ply_fn]]
+=== `ply`
+
+Subdivides a `datatable` and passes the resulting tables into an expression, 
+then merges the output.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
 
-* <<canvas-common-functions>> can run anywhere
-* <<canvas-server-functions>> can only execute on the server
-* <<canvas-client-functions>> must execute in a browser
+|`by` *** *†*
+|`string`
+|The column to subdivide
+
+|`expression` *** *†*
+
+Alias: `fn`, `exp`, `function`
+|`datatable`
+|An expression to pass into each resulting data table.
+Expressions must return a `datatable`. Use `as` to turn literals into datatables.
+Multiple expressions must return the same number of rows. If you need to return 
+a differing row count, pipe into another instance of `ply`.				
+If multiple expressions return the same columns, the last one wins.
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[pointseries_fn]]
+=== `pointseries`
+
+Turns a `datatable` into a point series model. Currently Canvas differentiates measure 
+from dimensions by looking for a <<canvas-tinymath-functions, Tiny Math function>>. If you enter a TinyMath 
+expression in your argument, Canvas treats that argument as a measure. Otherwise, it 
+is a dimension. Dimensions are combined to create unique keys. Measures are 
+then deduplicated by those keys using the specified TinyMath function.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`x`
+|`string`
+|The values along the X-axis
+
+|`y`
+|`string`
+|The values along the Y-axis
+
+|`color`
+|`string`
+|An expression to use in determining the mark's color
+
+|`size`
+|`string`
+|For elements that support it, the size of the marks
+
+|`text`
+|`string`
+|For use in charts that support it, the text to show in the mark
+|===
+
+*Returns:* `pointseries`
+
+
+[float]
+[[progress_fn]]
+=== `progress`
+
+Configure a progress element
+
+*Accepts:* `number`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`barColor`
+|`string`
+|Color of the background bar
+
+Default: `"#f0f0f0"`
+
+|`barWeight`
+|`number`
+|Thickness of the background bar
+
+Default: `20`
+
+|`font`
+|`style`
+|Font settings for the label. Technically you can stick other styles in here too
+
+Default: `{<<font_fn>> size=24 family="'Open Sans', Helvetica, Arial, sans-serif" color="#000000" align=center}`
+
+|`label`
+|`boolean`, `string`
+|Set `true`/`false` to show/hide label or provide a string to display as the label
```

Comment:
How about:

To show or hide labels, or provide a string to display as a label, use `true` or `false`.